src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -675,19 +682,18 @@ def _visit_map(self, node: gtir.FunCall) -> DataExpr:
                 )
                 assert isinstance(input_local_offset_provider, gtx_common.Connectivity)
                 if input_local_offset_provider.has_skip_values:
-                    skip_value_connectivities.append(
-                        (input_expr.local_offset, input_local_offset_provider)
-                    )
+                    skip_value_connectivities[input_expr.local_offset] = input_local_offset_provider
 
             input_nodes[input_node.data] = input_node
 
-        out, _ = self.sdfg.add_temp_transient((local_size,), dtype)
+        out, _ = self.sdfg.add_temp_transient((local_size,), dc_dtype)
         out_node = self.state.add_access(out)
 
         if len(skip_value_connectivities) != 0:
             # In case one or more of input expressions contain skip values, we use
             # the connectivity-based offset provider as mask for map computation.
-            local_offset, offset_provider = skip_value_connectivities[0]
+            # GT4Py guarantees that all input expressions have the same skip values.
```

Comment:
It is not about the value, it is about having them or not.

Reply:
Yes, it is also about the values. The skip values should appear at the same neighbor index for all arguments to map.